content/plotly/concepts/graph-objects/terms/mesh3d/mesh3d.md

```diff
@@ -0,0 +1,85 @@
+---
+Title: '.Mesh3d()'
+Description: 'A brief description.'
+Subjects:
+- 'Data Visualization'
+- 'Data Visualization'
+- '3D Graphing'
+- 'Plotly'
+Tags:
+- 'plotly'
+- 'mesh'
+- '3D-plotting'
+
+CatalogContent:
+
+- 'https://www.codecademy.com/learn/paths/computer-science'
+
+- 'https://www.codecademy.com/learn/learn-python-3'
+
+- 'https://plotly.com/python/getting-started/'
+
+- 'https://plotly.github.io/plotly.py-docs/generated/plotly.graph_objects.Mesh3d.html'
+
+
+---
```

Comment:
```suggestion
---
Title: '.Mesh3d()'
Description: 'Creates 3D triangular mesh plots for visualizing 3D surfaces or volumetric data.'
Subjects:
- 'Data Science'
- 'Data Visualization'
Tags:
- 'Charts'
- 'Data'
- 'Plotly'
CatalogContent:
- 'learn-python-3'
- 'paths/computer-science'
---
```

Reply:
The tags, subjects and catalog are needed to be added from here: [tags](https://github.com/Codecademy/docs/blob/main/documentation/tags.md), [subjects](https://github.com/Codecademy/docs/blob/main/documentation/subjects.md) and c[atalog content](https://github.com/Codecademy/docs/blob/main/documentation/catalog-content.md)

Also, a short description of the term has to be added, I've added one, you can change it, that is just for reference.